ubuntu_image/assertions.py

```diff
@@ -24,7 +24,8 @@ def from_string(cls, text):
         # XXX: This is a temporary stop-gap. The proper way to do this is to
         # invoke yet-unimplemented go executable that reads an assertion on
         # stdin, validates it and outputs the same assertion as JSON on stdout.
-        records = load_rfc822_records(StringIO(text))
+        (fields, signature) = text.rsplit('\n\n', 1)
+        records = load_rfc822_records(StringIO(fields))
```

Comment:
Actually, I think we should eventually ditch this stuff and use the stdlib email package parser, with a policy for any field value assertions if necessary.  That is, if we still end up needing any of this so... later!